src/Page/ExpressivePage.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+namespace Zend\Navigation\Page;
+
+use Zend\Expressive\Router\Exception\RuntimeException as RouterException;
+use Zend\Expressive\Router\RouteResult;
+use Zend\Expressive\Router\RouterInterface;
+use Zend\Navigation\Exception;
+
+/**
+ * Class ExpressivePage
+ */
+class ExpressivePage extends AbstractPage
+{
+    /**
+     * Route name
+     *
+     * @var string
+     */
+    private $routeName;
+
+    /**
+     * Route parameters
+     *
+     * @var array
+     */
+    private $routeParams = [];
+
+    /**
+     * @var array
+     */
+    private $queryParams = [];
+
+    /**
+     * @var RouteResult
+     */
+    private $routeResult;
+
+    /**
+     * @var RouterInterface
+     */
+    private $router;
+
+    /**
+     * @var string
+     */
+    private $hrefCache;
+
+    /**
+     * @inheritdoc
```

Comment:
https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#62-using-the-inheritdoc-inline-tag-to-augment-a-description